frontend/lib/api/form/pollingstation/PollingStationFormController.tsx

```diff
@@ -446,6 +447,17 @@ export function PollingStationFormController({
     status.current = "deleted";
   };
 
+  const finaliseDataEntry = async () => {
+    status.current = "finalising";
+    const response = await client.postRequest(request_path + "/finalise");
+    if (response.status !== ApiResponseStatus.Success) {
+      console.error("Failed to finalise data entry", response);
+      status.current = "idle";
+      setError(response.data as ApiResponseErrorData);
+    }
+    status.current = "finalised";
```

Comment:
Shouldn't this be an explicit `else`? Because the if does not have a return and the state will always be set to `finalised` now I believe.

Reply:
Good catch, this was missing a `throw` that I've added back.